traffic_portal/app/src/common/modules/form/user/form.user.tpl.html

```diff
@@ -108,15 +108,13 @@
             <div class="form-group" ng-if="user.newUser" ng-class="{'has-error': hasError(userForm.registrationSent), 'has-feedback': hasError(userForm.registrationSent)}">
                 <label class="control-label col-md-2 col-sm-2 col-xs-12">Registration Sent</label>
                 <div class="col-md-10 col-sm-10 col-xs-12">
-                    <input name="registrationSent" type="text" class="form-control" ng-model="user.registrationSent" ng-maxlength="128" required ng-disabled="!settings.isNew">
-                    <small class="input-error" ng-show="hasPropertyError(userForm.registrationSent, 'required')">Required</small>
-                    <small class="input-error" ng-show="hasPropertyError(userForm.registrationSent, 'maxlength')">Too Long</small>
+                    <input name="registrationSent" type="text" class="form-control" ng-model="user.registrationSent" ng-maxlength="128" readonly>
                     <span ng-show="hasError(userForm.registrationSent)" class="form-control-feedback"><i class="fa fa-times"></i></span>
                 </div>
             </div>
 
             <div class="modal-footer">
-                <button ng-if="user.newUser" type="button" class="btn btn-default" ng-click="resendRegistration(user)">Resend Registration</button>
+                <button ng-if="!settings.isNew" type="button" class="btn btn-default" ng-click="resendRegistration(user)">Resend Registration</button>
```

Comment:
this was fine the way it was.

Reply:
:(

Reply:
yeah, so the dumb newUser field basically translates to "user was created via a registration" which means the "Resend Registration" button should be displayed.